Time Like Splitting Functions (#232)
NNPDF/eko: doc/source/theory/TimeLike.rst

analysis of experimental data involving different processes for their reliable
determination. This makes the |FF| similar to |PDF| as both rely
on similar factorization theorems and, thus, on similar |RGE|.
In practice, the relevant Feynman diagrams can indeed be related by a crossing
symmetry which in turn means certain Mandelstam variables become for |FF|
time-like instead of space-like.
The relevant setting in the operator card is thus called ``time_like = True``.

We implement the time-like |DGLAP| anomalous dimensions up to |NNLO| in :class:`~ekore.anomalous_dimensions.unpolarized.time_like`.
The implementation for the |LO| and |NLO| splitting functions is based on :cite:`Mitov:2006wy, Gluck:1992zx` and the implementation for
the |NNLO| splitting functions is based on :cite:`Mitov:2006ic, Moch:2007tx, Almasy:2011eq`.
Supplying new anomalous dimensions and new matching conditions is the only change required for the eko program (e.g. the
solution strategies are unaffected).

The time-like splitting functions have been benchmarked using select |FF| sets from |LHAPDF|. In addition, they have been tested 

> against routines from |MELA|, which is an alternative Mellin Space evolution code, available at https://github.com/vbertone/MELA.

Reviewer: you have to define the abbreviation for Mela - and the existing abbreviation for lhapdf is with lower case... please check this page also live! you can do so by `poe docs` and then `poe docs-server` for more details also check the `pyproject.toml`

> The time-like splitting functions have been benchmarked using select |FF| sets from |LHAPDF|. In addition, they have been tested 
> against functions from |MELA|, which is an alternative Mellin Space evolution code, available at https://github.com/vbertone/MELA.

Reviewer: ```suggestion
The time-like evolution has been benchmarked using selected |FF| sets from |LHAPDF|. In addition, the anomalous dimensions have been tested 
against functions from |MELA|, which is an alternative Mellin Space evolution code, available at https://github.com/vbertone/MELA.
```
